### PR TITLE
Fix SVG Blockloader Breaking in Chrome

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

The skeleton loader is and svg which requires pixel values.
This PR ensure px values are used since chrome is strict and was breaking.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Screenshots

Issue:
Screen Shot 2019-09-09 at 2 32 23 PM

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: .bumpversion.cfg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.1.8
+current_version = 0.1.9
```
Comment (DavidLemayian) [inline .bumpversion.cfg]: @karimkawambwa right now bumpversion should only occur in master branch directly.

Comment (DavidLemayian) [reply .bumpversion.cfg]: This is to avoid conflicts and for it to happen just before / as part of publish.

File: src/BlockLoader.js [hunk 2]
```diff
@@ -18,12 +18,12 @@ BlockLoader.propTypes = {
...
-  height: PropTypes.oneOfType([PropTypes.string, PropTypes.number])
+  width: PropTypes.number,
+  height: PropTypes.number
 };
 
 BlockLoader.defaultProps = {
   children: null,
-  width: '100%',
-  height: '100%'
+  width: -1,
```
Comment (kilemensi) [inline src/BlockLoader.js]: A comment may be for why  `-1` and not `undefined`?

File: src/ContentLoader.js [hunk 3]
```diff
@@ -24,15 +40,20 @@ export default function CustomContentLoader({
...
+  height: PropTypes.number
 };
 
 CustomContentLoader.defaultProps = {
-  width: '100%',
-  height: '100%'
+  id:
+    Math.random()
+      .toString(36)
+      .substring(2) + Date.now().toString(36),
```
Comment (kilemensi) [inline src/ContentLoader.js]: We are generating these random values in a few places now, shouldn't we just include a dependency to help us with these? [shortid](https://github.com/dylang/shortid) or something similar?

File: src/BlockLoader.js (hunk 2 above)
Comment (karimkawambwa) [reply src/BlockLoader.js]: good question hehe, I don't know 🤔

File: src/ContentLoader.js (hunk 3 above)
Comment (karimkawambwa) [reply src/ContentLoader.js]: 🤔

**Pull request merged**